config/var_qa.env

```diff
@@ -2,10 +2,11 @@ DOCS_BASE_PATH=/api
 DOCS_EXTERNAL_DESCRIPTION="Documentation for using the Platform APIs"
 DOCS_EXTERNAL_URL=https://docs.google.com/document/d/1p3q9OT9latXqON20WDh4CNPxIShUunfGgqT163r-Caw/edit?usp=sharing
 DOCS_HOST=qa-platform.nypl.org
+DOCS_INFO_DESCRIPTION="Endpoints require an OpenID Connect token from the NYPL OAuth server and interact with NYPL Data Streams."
```

Comment:
We had thought that since DOCS_INFO_DESCRIPTION  and DOCS_INFO_TITLE were common to everything, they would be in var_app only, is that not correct?

Reply:
That's a good point! I moved them to `global.env` which is the way to accomplish them in the current Microservice Starter.